lambdas/stac-gen/handler.py

```diff
@@ -30,14 +31,13 @@ def upload_stac_to_s3(stac_dict):
             f"stac_item_queue/{fname}.json",
         )
         print("File uploaded to s3")
-        return f"s3://{output_bucket}/{collection}/{filename}"
+        return f"s3://climatedashboard-data/stac_item_queue/{fname}.json"
```

Comment:
I think this should be configured as an environment variable

Reply:
I agree, lets add this in a separate PR where we split up the workflow in to separate stacks. getting complicated to do all this configuration within this PR

Reply:
Sounds good to me!